gapic/src/main/com/google/gapid/perfetto/views/State.java

```diff
@@ -68,6 +73,7 @@
   private final AtomicInteger lastSelectionUpdateId = new AtomicInteger(0);
   private HashMultimap<Long, Long> selectedThreads;     // upid -> utids
   private TimeSpan highlight = TimeSpan.ZERO;
+  private TreeMap<Double, Long> flags;
```

Comment:
If you could provide a comment briefly tell what's the key-value pair, like in L74, it would be great.